slices/fonts-dejavu-core.yaml

```diff
@@ -0,0 +1,35 @@
+package: fonts-dejavu-core
```

Comment:
It probably depends on `fonts-dejavu-mono` too, right? Do we need the mono fonts?